features/oauth.feature

```diff
@@ -145,11 +152,11 @@ Feature: Using OAuth for 2-legged OAuth flow (client credentials)
       "MJ-TemplateLanguage": true,
       "Recipients": [
         {
-          "Email": "jp@test.com",
-          "Name": "Jean-Pierre d'Artagnan",
+      "Email": "jp@test.com",
+          "Name": "Jean-Pierre Durand",
           "Vars": {
             "first_name": "Jean-Pierre",
-            "activation_link": "http:\/\/test.enmarche.code\/inscription\/finaliser\/@string@\/@string@?redirect_uri=https%3A\/\/en-marche.fr\/callback&client_id=f80ce2df-af6d-4ce4-8239-04cfcefd5a19"
+            "activation_link": "http:\/\/test.enmarche.code\/inscription\/finaliser\/@string@\/@string@"
```

Comment:
The scenario name is  `Scenario: Register a user with callback URI`.
So the feature is broken at this point or this test is now pointless because requirements changed.
What's the answer ?